tests/main/exitcodes/task.yaml

```diff
@@ -0,0 +1,31 @@
+summary: Checks for snap exit codes
+
+systems: [ubuntu-1*, ubuntu-2*]
+
+restore: |
+    umount /usr/bin/mksquashfs || true
```

Comment:
Could I encourage you to try `tests.cleanup`, then you don't need the `|| true` part anymore.